docs/user-guide.md

```diff
@@ -0,0 +1,72 @@
+
+![Verify Proofs](../resources/screenshots/first.png)
+
+### Running Proofs
+
+
+You can then run your proofs using the test tree by clicking the play button beside the harness that was automatically picked up by the Kani Extension. Once you run the proof using the extension, you are shown an error message if the proof has failed. We are then presented with two buttons, to generate the full report for the harness or to run concrete playback to generate unit tests with the counter examples loaded.
+
+![Image: run proof.gif](../resources/screenshots/run%20proof.gif)
+
+### View and Debug Proofs through the Trace
+
+You can generate the unit test with the counter examples by clicking on the **Run Concrete Playback for (your harness name)`** Button that appears through a blue link on the error banner.
+
+![Image: generate counter example.gif](../resources/screenshots/generate%20counter%20example.gif)
+
+You can see that the source is annotated with two buttons that hover over the unit test generated called → `Run Test (Kani) | Debug Test (Kani)` which allow you to run and debug the test just like any other rust unit test.
+
+### Running Kani-generated test
+
+Clicking the unit test using the `Run Test (Kani)` button, runs the unit test generated by concrete playback.
+
+![Run Concrete Test](../resources/screenshots/third.png)
+
+### Debug Kani unit test
+
+By setting breakpoints and clicking the debug test (kani) button, you are taken into the debugger panel which allows you to peer into the counter examples or values for which the assert fails.
+
+![Image: show debugging.gif](../resources/screenshots/show%20debugging.gif)
+
+
+(**NOTE TO READER -** Try reloading page to view the gif).
+
+
+You can then use the debugger controller to step through, into, out of, replay and also change values on the trace panel on the left for interactive debugging.
+
+### Generate HTML report
+
+By clicking the view report button in the error message, you can view the trace for the harness in a rendered HTML report.
+
+![Generate Report](../resources/screenshots/generate-report.png)
+
+### View Report in-window
+
+You can click on the `Preview in Editor` to view the HTML trace within vscode. It should look like this -
+
+![Generate Report](../resources/screenshots/view-report.png)
+
+
+### View Kani Log
+
+For every test run, you can view full output from kani logged into the output channel as a text file. To view the log, open the output channel, and click on the channels drop down list to view a channel called `Output (Kani): ...`
+
+![Generate Report](../resources/screenshots/view-output.png)
+
+
+### Other features
+
+Some other features include -
+
+1. Verify entire crate at once or run your file organized proofs together.
+2. Keybindings, shortcuts, configurations
+3. View output channel
+4. View time based logs of the verification
```

Comment:
Many minor issues throughout this file:
 * "rust" instead of "Rust" (and similar ones)
 * "-" isn't a good replacement for ":" (or at least I'm not used to it)
 * Don't know what some features are about, or if they're worth mentioning.

Reply:
I think I can just remove the entire section.